components/molecule/accordion/src/_settings.scss

```diff
@@ -0,0 +1,10 @@
+$bd-accordion-tab-container: 1px $c-gray-lightest groove !default;
+$bd-accordion-tab-container-none: 0 $c-gray-lightest groove !default;
+$bgc-accordion-tab-container: $c-white !default;
+$c-accordion-tab-icon: $c-primary !default;
+$c-accordion-tab-text: $c-text-base !default;
+$h-accordion-tab-btn: 40px !default;
+$mh-accordion-tab-btn: 24px !default;
```

Comment:
What about using as default variable a sui-theme variable?

Reply:
I can't find any variable for height or maximum-height in sui-theme. Should I create a new one?

Reply:
:D Sorry, I thought that mh was horizontal margin. :)